src/slic3r/GUI/Widgets/SwitchButton.cpp

```diff
@@ -9,8 +9,8 @@
 
 SwitchButton::SwitchButton(wxWindow* parent, wxWindowID id)
 	: wxBitmapToggleButton(parent, id, wxNullBitmap, wxDefaultPosition, wxDefaultSize, wxBORDER_NONE | wxBU_EXACTFIT)
-	, m_on(this, "toggle_on", 16)
-	, m_off(this, "toggle_off", 16)
+	, m_on(this, "toggle_on", 16, false, false, true)
+	, m_off(this, "toggle_off", 16, false, false, true)
```

Comment:
use the legacy bmp format. using bitmap bundles causes issues (I specifically noticed with the advanced toggle).